dnstracer: add page (#8707)
tldr-pages/tldr: pages/linux/dnstracer.md

# dnstracer

> The dnstracer command determines where a DNS gets its information from.

> > More information: <http://www.mavetju.org/unix/dnstracer.php>.

Reviewer: ```suggestion
> More information: <https://manned.org/dnstracer>.
```

Here, we prefer using the `dnstracer`'s official [`man`](https://manned.org/man) documentation as hosted in manned.org. See #5143 for more info.

Author: Fine with me. I had linked the home page, because it was referred on the Kali tools page. `man` seems more sensible though.

- Find out where your local DNS got the information on www.example.com:

`dnstracer {{www.example.com}}`

> - Start with a specific DNS that you already know:

Reviewer 2: ```suggestion
- Start with a [s]pecific DNS that you already know:
```

`dnstracer -s {{dns.example.org}} {{www.example.com}}`

- Only query IPv4 servers:

`dnstracer -4 {{www.example.com}}`

> - [r]etry each request 5 times on failure:

Reviewer 2: Hey, Sorry for the request can you revert this change?
```suggestion
- Retry each request 5 times on failure:
```
As `r` means `retries` your original version of the command description is fine as the mnemonic wouldn't be applicable here.

Author: Of course, makes sense. Personally, I'd be fine with either, but it's of course preferable to be consistent with other pages.

> - Retry each request 5 times on failure:

Reviewer 2: ```suggestion
- [r]etry each request 5 times on failure:
```

> `dnstracer -r 5 {{www.example.com}}`

Reviewer 3: ```suggestion
`dnstracer -r {{5}} {{www.example.com}}`
```

- Display all steps during execution:

`dnstracer -v {{www.example.com}}`

> - Display an overview of all received answers after execution:

Reviewer 2: ```suggestion
- Display an [o]verview of all received answers after execution:
```

Author: Understood, will keep in mind for future commits 🙂
